### DISCUSSION TITLE
Is Faststore production ready ?

### DISCUSSION THREAD

**Discussion by xXDeonoXx** [vtex/faststore#1825] [Q&A]
Hi, i would like to know if Faststore is stable and ready enough to be used in a real e-commerce.

I see that the docs are being updated daily/weekly and the V1 was released recently.

We are using the NextJS starter as the starter point and we saw some GraphQL queries in the pages as fragments not being used and some other things like incomplete docs that concerned us.

**Comment by viniciusmlira** [comment]
Hi, @xXDeonoXx!

Sorry for the delay, but we have updates regarding this topic!

FastStore is ready to use and it's on v2.2 right now. 

So far we still have some conditions to start using it. 

- For going live with FastStore:

Partners/customers must reach their VTEX interface (Solution Architect, CX, CSM, or similar) and apply to our FastStore Early Access Program.

Customers/partners are going to be assisted by VTEX while developing their storefront, and must deliver some feedback about the overall experience. The program is running now and should end by the end of the year.

Then we're going to announce publicly how to start using FastStore with more flexibility.

- For browsing/experiencing FastStore without going live

If you are in a VTEX partner and are curious about FastStore, you can request a experimental environment to explore our product. If you wanna check our official documentation, just check out this link [here](https://www.faststore.dev/)

Just reach out your contact at VTEX and we can provide you the proper environment, ok?

Tks for the questions!

best,

@FastStore team
